webapp/src/Controller/API/UserController.php

```diff
@@ -216,7 +213,9 @@ public function addTeamsAction(Request $request): string
      *     response="200",
      *     description="Returns a (currently meaningless) status message.",
      * )
+     *
      * @throws BadRequestHttpException
+     * @throws Exception
```

Comment:
why do we need to annotate this with the more generic `Exception`?

Reply:
`importTsv` throws it. Which it doesn't really have to so maybe we should not do this.

Reply:
Done